docs/lifecycle.md

```diff
@@ -108,6 +128,23 @@ See AWS specs for details on the protocol format.
 Those calls essentially manage bucket attributes related to lifecycle
 behavior, which are stored as part of bucket metadata.
 
+#### Transition Policies support in CloudServer
+
+Transition policies rely on the *preferred read location* for an
+object.
+
+We introduce an optional per-object *preferred read location*. If set,
+it is used to determine which location to read the object from on a
+GET request. It overrides the *preferred read location* defined in the
+bucket replication configuration (if any).
+
+The attribute may be present in the `replicationInfo` section of
+object metadata, as a `"preferred": "<location>"` property, where
+`<location>` is one of the replication targets listed in
+`replicationInfo` that should be in `COMPLETED` state in order to make
```

Comment:
I'd put a period after `replicationInfo` and start next sentence with 'It should be in the 'COMPLETED' state...'